src/components/ProjectListItemCollapsable/ProjectListItemCollapsable.tsx

```diff
@@ -28,6 +28,7 @@ interface ProjectListItemCollapsableProps extends Omit<ComponentProps<typeof Tre
     goals?: ReactNode;
     children?: React.ReactNode;
     onClick?: MouseEventHandler<HTMLElement>;
+    inner?: boolean;
```

Comment:
Why not to declare size as is?

Reply:
'Cause this not specific prop for inner component, what if we wants use this prop for the other inner component styling?

Reply:
I see, but inner prop means nothing. Also it is not extendable in case of more than 2 variants of title. U can use titleSize instead size. It's much more specific and obvious.